bundles/org.openhab.binding.buienradar/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="buienradar"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="rain_forecast">
+		<label>Rain forecast</label>
+		<description>Does a simple rain forecast at the specified longitude/latitude of
+		two hours, in 5 minute increments using the buienradar.nl webservice. </description>
+
+		<channels>
+			<channel id="forecast_0" typeId="forecast_0" />
+			<channel id="forecast_5" typeId="forecast_5" />
+			<channel id="forecast_10" typeId="forecast_10" />
+			<channel id="forecast_15" typeId="forecast_15" />
+			<channel id="forecast_20" typeId="forecast_20" />
+			<channel id="forecast_25" typeId="forecast_25" />
+			<channel id="forecast_30" typeId="forecast_30" />
+			<channel id="forecast_35" typeId="forecast_35" />
+			<channel id="forecast_40" typeId="forecast_40" />
+			<channel id="forecast_45" typeId="forecast_45" />
+			<channel id="forecast_50" typeId="forecast_50" />
+			<channel id="forecast_55" typeId="forecast_55" />
+			<channel id="forecast_60" typeId="forecast_60" />
+			<channel id="forecast_65" typeId="forecast_65" />
+			<channel id="forecast_70" typeId="forecast_70" />
+			<channel id="forecast_75" typeId="forecast_75" />
+			<channel id="forecast_80" typeId="forecast_80" />
+			<channel id="forecast_85" typeId="forecast_85" />
+			<channel id="forecast_90" typeId="forecast_90" />
+			<channel id="forecast_95" typeId="forecast_95" />
+			<channel id="forecast_100" typeId="forecast_100" />
+			<channel id="forecast_105" typeId="forecast_105" />
+			<channel id="forecast_110" typeId="forecast_110" />
+			<channel id="forecast_115" typeId="forecast_115" />
+		</channels>
+
+		<config-description>
+			
+			<parameter name="latitude" type="decimal" required="true">
```

Comment:
There is standard support for this in openHAB. Use `<context>location</location>` with a single parameter of type String. See `openweathermap` binding for how this works. It makes it possible to select the location on a map in the ui.

Reply:
Fixed.